packages/compiler/src/myst.ts

```diff
@@ -36,5 +59,95 @@ export function parsePanel(contents: string): Array<MystCard> {
     parsedCard.body = body;
     splitCards.push(parsedCard);
   }
-  return splitCards;
+
+  let myPanel: MystPanel = { cards: splitCards };
+  if (panelStyle) {
+    myPanel = { ...myPanel, style: panelStyle };
+  }
+  return myPanel;
+}
+
+export function parseStyling(contents: string): [string, string] {
+  let yamlBlock = "";
+  let returnContents = "";
+  const contentLines = contents.split("\n");
+  if (contents.startsWith(":")) {
+    const yamlLines: Array<string> = [];
+    while (contentLines) {
+      if (!ltrim(contentLines[0]).startsWith(":")) {
```

Comment:
I believe you can replace `ltrim` with the `.trimLeft` builtin

```suggestion
      if (!contentLines[0].trimLeft().startsWith(":")) {
```

Reply:
New version of this is ```.trimStart()``` apparently.

Reply:
https://github.com/irydium/irydium/blob/ae9b5c1fc923c5c0f92b5ae15ed3782011f70ada/packages/compiler/src/myst.ts#L80